substrate/frame/src/lib.rs

```diff
@@ -403,17 +418,27 @@ pub mod runtime {
 		pub use sp_version::NativeVersion;
 
 		/// Macro to implement runtime APIs.
-		pub use sp_api::impl_runtime_apis;
+		pub use sp_api::{decl_runtime_apis, impl_runtime_apis};
 
 		// Types often used in the runtime APIs.
-		pub use sp_core::OpaqueMetadata;
+		pub use sp_core::{
+			offchain::{testing::TestOffchainExt, OffchainDbExt, OffchainWorkerExt},
```

Comment:
would not re-export testing primitives directly

```suggestion
			offchain::testing,
```